### DISCUSSION TITLE
Can I know how much size is available for containers?

### DISCUSSION THREAD

**Discussion by peakedshout** [fyne-io/fyne#4603] [Q&A] [answered]
### Checklist

- [ ] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

New containers are always displayed at the smallest size, which seems strange when the user stretches the window and cannot make proper adjustments to the components.
I currently use most boxes for nesting, and I need to display the list internally, because the minimum size is displayed by default, and the list cell content cannot be displayed, but if the size is preset, when the window is stretched, I can't make timely feedback adjustments

### Is it possible to construct a solution with the existing API?

No, I've tried different layouts, but none of them worked well

### Describe the solution you'd like to see.

At the very least, I want to get how much space the container can get in the current window, so I can change the size of the component.

**Comment by Jacalz** [comment]
Please use discussions for questions. Issues are for feature requests and bugs. Will convert this for you.

**Comment by andydotxyz** [comment, marked as answer]
This is what the container Layout does. If you want to handle the space manually then set up a custom layout. https://docs.fyne.io/extend/custom-layout

**Reply by peakedshout** [reply]
No, I just want to know how much space is left and I need to fill it up

**Reply by peakedshout** [reply]
Sorry, I tried it and it seems that the layout does have a way of updating the size with a callback, and it looks like I should write a layout similar to a canvas.
But I'm having a new problem:https://github.com/fyne-io/fyne/discussions/4607
This led me to think there was something wrong with the layout, but I tried it many times and found that the way the composition was combining was causing the components to not be displayed, and I didn't know what was going on

**Reply by Jacalz** [reply]
I can recommend to have a look at https://m.youtube.com/watch?v=LWn1403gY9E&embeds_referring_euri=https%3A%2F%2Fdocs.fyne.io%2F&source_ve_path=OTY3MTQ&feature=emb_imp_woyt.
